Content.Server/Blob/BlobObserverSystem.cs

```diff
@@ -0,0 +1,862 @@
+            targetTile.GridIndices.Offset(Direction.West),
+            targetTile.GridIndices.Offset(Direction.North),
+            targetTile.GridIndices.Offset(Direction.South)
+        };
+
+        if (!adjacentTiles.Any(indices =>
+                grid.GetAnchoredEntities(indices).Any(ent => HasComp<BlobTileComponent>(ent))))
+            return;
+        var cost = blobCoreComponent.NormalBlobCost;
+        if (targetTileEmplty)
+        {
+            cost *= 2;
+        }
+
+        if (!_blobCoreSystem.TryUseAbility(uid, observerComponent.Core.Value, blobCoreComponent, cost))
+            return;
+
+        if (targetTileEmplty)
+        {
+            var plating = _tileDefinitionManager["Plating"];
+            var platingTile = new Tile(plating.TileId);
+            grid.SetTile(location, platingTile);
+        }
+
+        _blobCoreSystem.TransformBlobTile(null,
+            observerComponent.Core.Value,
+            blobCoreComponent.NormalBlobTile,
+            location,
+            blobCoreComponent,
+            transformCost: cost);
+    }
+
+    private void OnStartup(EntityUid uid, BlobObserverComponent component, ComponentStartup args)
+    {
+        var helpBlob = new InstantAction(
+            _proto.Index<InstantActionPrototype>("HelpBlob"));
+        _action.AddAction(uid, helpBlob, null);
+        var swapBlobChem = new InstantAction(
+            _proto.Index<InstantActionPrototype>("SwapBlobChem"));
+        _action.AddAction(uid, swapBlobChem, null);
+        var teleportBlobToCore = new InstantAction(
+            _proto.Index<InstantActionPrototype>("TeleportBlobToCore"));
+        _action.AddAction(uid, teleportBlobToCore, null);
+        var teleportBlobToNode = new InstantAction(
+            _proto.Index<InstantActionPrototype>("TeleportBlobToNode"));
+        _action.AddAction(uid, teleportBlobToNode, null);
+        var createBlobFactory = new WorldTargetAction(
+            _proto.Index<WorldTargetActionPrototype>("CreateBlobFactory"));
+        _action.AddAction(uid, createBlobFactory, null);
+        var createBlobResource = new WorldTargetAction(
+            _proto.Index<WorldTargetActionPrototype>("CreateBlobResource"));
+        _action.AddAction(uid, createBlobResource, null);
+        var createBlobNode = new WorldTargetAction(
+            _proto.Index<WorldTargetActionPrototype>("CreateBlobNode"));
+        _action.AddAction(uid, createBlobNode, null);
+        var createBlobbernaut = new WorldTargetAction(
+            _proto.Index<WorldTargetActionPrototype>("CreateBlobbernaut"));
+        _action.AddAction(uid, createBlobbernaut, null);
+        var splitBlobCore = new WorldTargetAction(
+            _proto.Index<WorldTargetActionPrototype>("SplitBlobCore"));
+        _action.AddAction(uid, splitBlobCore, null);
+        var swapBlobCore = new WorldTargetAction(
+            _proto.Index<WorldTargetActionPrototype>("SwapBlobCore"));
+        _action.AddAction(uid, swapBlobCore, null);
```

Comment:
This should be a list or two on the component instead of copy-pasted hardcoding.